data/analytics/events.yml

```diff
@@ -673,7 +673,43 @@
           value: Get emails from GOV.UK
         - name: type
           value: email subscription
-
+- name: filter
+  description: Interactions that can be conducted after filtering the content using CSS instead of HTTP or API requests.
+  events:
+  - name: filter search
+    implemented: true
+    priority: low
+    description: When a user searches for a link on a page, and the search box is one that filters all the results using CSS instead of HTTP or API requests.
```

Comment:
Suggestion: `When a search box is used to filter a long list of things on a page, by showing relevant matches using JavaScript, rather than fetching a new page of results from the server`

Reply:
@andysellick I was hesitant to just say JavaScript, since our finder searches use client side JavaScript to update search results. Given this, should I still use your suggestion?

Reply:
How about `When a search box is used to filter a long list of things on a page, by hiding irrelevant matches, rather than fetching a new page of results from the server` ?

Reply:
@andysellick  Sounds good 👍 Have updated both descriptions, should be ready for another review

Reply:
Updated now!